pkg/asset/installconfig/azure/validation.go

```diff
@@ -200,9 +214,12 @@ func validateInstanceTypes(client API, ic *types.InstallConfig) field.ErrorList
 			if ultraSSDCapability == "" {
 				ultraSSDCapability = defaultUltraSSDCapability
 			}
+			if vmNetworkingType == "" {
+				vmNetworkingType = defaultVMNetworkingType
+			}
```

Comment:
Do compute nodes default to Accelerated Networking is true? I thought control plane defaulted to accelerated networking true but compute nodes false, but I could be wrong.

Reply:
This was a discussion that @staebler and I had in slack. We discussed that we should move the default to true for all nodes types since the default for nodes also includes 4 vCPU.